Makefile

```diff
@@ -1,18 +1,23 @@
 .PHONY: start
 
 # Read values as needed from .env
+#
+# Optionally pass 'environment=<something>' to load env vars from '.env-something' instead
+# Useful to separate i.e. dev from testing databases
```

Comment:
[very minor] usually it is `.env.something`, like `.env.local` in most of the tooling that support it